app/models/dbModels/article.js

```diff
@@ -53,6 +53,10 @@ const ArticleSchema = new Schema(
     collection: "articles",
   }
 );
+
+// Add a text index for title and articleContent.content
+ArticleSchema.index({ title: "text", "articleContent.content": "text" }, { name: "article_text_index" });
+
```

Comment:
Is this necessary since the index is set on MongoDB Atlas? 

It could be fine for prod, just on a local MongoDB service I'm not sure if this would cause issues